openapi.json

```diff
@@ -4762,15 +4762,6 @@
           "example": {
             "description": "A free-form property to include an example for this argument. To represent examples that cannot be naturally represented in JSON, a string value can be used to contain the example with escaping where necessary."
           }
-        },
```

Comment:
Having an example would be good nevertheless. 

Could you add something like the following snippet, please?

```json
{
  "description": "A percentage between 0 and 100.",
  "required": true,
  "minimum": 0,
  "maximum": 100
}
```